src/newWizard/WizardStep.tsx

```diff
@@ -31,6 +31,7 @@ export type WizardStepProps = {
     navigationClasses?: ReadonlyArray<string>;
     navigable?: boolean;
     type?: WizardStepType;
+    isApplying?: boolean;
```

Comment:
is it required here? or is it for future use?

Reply:
@girishdudhe This is required as these props shared with wizard navigation component as well